MobiFlight/MidiBoard/MidiInputDefinition.cs

```diff
@@ -40,7 +41,7 @@ public class MidiInputDefinition
         /// <summary>
         /// The midi message ids. Possible value range from 0 to 127. Required.
         /// </summary>
-        public byte[] MessageIds;
+        public List<byte> MessageIds;
```

Comment:
This needs to be a `List` instead of a `byte[]` so when the object is converted to JSON for schema validation the `byte[]` doesn't get serialized as a `string`.